helm_deploy/prisoner-content-hub-backend/templates/drupal-post-install-hook.yaml

```diff
@@ -18,6 +18,24 @@ spec:
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     spec:
       restartPolicy: Never
+  {{ if .Values.dbRefresh.enabled }}
+      initContainers:
+        - name: db-refresh
+          image: "ghcr.io/ministryofjustice/hmpps-mysql-tools:latest"
+          command:
+            - /bin/entrypoint.sh
+          volumeMounts:
+            - name: db-refresh-script
+              mountPath: /bin/entrypoint.sh
+              readOnly: true
+              subPath: entrypoint.sh
+{{ include "db-backup.envs" . | nindent 10 }}
```

Comment:
We also run the db refresh on deployment, as part of the post-install-hook.  Note this only runs where   This ensures we are always testing where `.Values.dbRefresh.enabled` is set, which is just on dev and staging.
Running the db import at this point will slow down the deployment process, but ensures we are always testing our changes with the most up to date database.